lighthouse-core/scoring.js

```diff
@@ -56,9 +62,12 @@ class ReportScoring {
         // will still be included in the final report json and displayed in the report as
         // "Not Applicable".
         const result = resultsByAuditId[member.id];
-        if (result.scoreDisplayMode === Audit.SCORING_MODES.NOT_APPLICABLE) {
+        if (result.scoreDisplayMode === Audit.SCORING_MODES.NOT_APPLICABLE ||
+            result.scoreDisplayMode === Audit.SCORING_MODES.INFORMATIVE ||
+            result.scoreDisplayMode === Audit.SCORING_MODES.MANUAL) {
           member.weight = 0;
         }
+
```

Comment:
I think these members are `auditRef`s now if you want to do a drive by in this function :)